src/schema/v2/auction_result.ts

```diff
@@ -109,6 +110,10 @@ const AuctionResultType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ external_url }) => external_url,
     },
+    boughtIn: {
+      type: GraphQLBoolean,
+      resolve: ({ bought_in }) => bought_in,
```

Comment:
#minor Is this `resolve` function needed? is it because of camel case vs snake case?

Reply:
🐈 probably, otherwise i think it would look for a key called `boughtIn` in the response.